.github/workflows/build.yml

```diff
@@ -0,0 +1,70 @@
+name: build
+
+on:
+  push:
+    branches: [ dev, rel/* ]
+    tags: [ '*' ]
+  pull_request:
+    branches: [ dev, rel/* ]
+
+jobs:
+  build:
+    name: ${{ matrix.os }}
+    runs-on: ${{ matrix.os }}
+
+    strategy:
+      fail-fast: false
+      matrix:
+        os: [ macos-latest, ubuntu-latest, windows-latest ]
+        include:
+          - os: macos-latest
+            os_name: macos
+          - os: ubuntu-latest
+            os_name: linux
+          - os: windows-latest
+            os_name: windows
+
+    steps:
+
+    - name: Checkout code
+      uses: actions/checkout@v2
+
+    - name: Setup .NET Core SDK
```

Comment:
I guess it's only required for to be able to use `dotnet nuget`, as Arcade is supposed to provide us with the .NET Core SDK for the build part, right?

Reply:
I think arcade would use the same scripts to download this anyway, but my understanding is that the action does some caching stuff so that it might end up being quicker.

That said, this is just copy-paste from my own non-Arcade stuff, so I could also just remove it.

Reply:
Wait, I’m an idiot. I misread `dotnet NuGet` as `dotnet restore`, so actually, yes it is needed! 😅